charts/federation-v2/README.md

```diff
@@ -36,17 +36,30 @@ cluster registry:
 $ kubectl create ns kube-multicluster-public
 ```
 
-Install the chart with the release name `federation-v2`:
+Next, add the chart to your local repository.
+```bash
+$ helm repo add federation-v2 https://raw.githubusercontent.com/kubernetes-sigs/federation-v2/add-chart-repo-index/charts/
```

Comment:
- From subsequent commands the repo name should be `fedv2-charts` instead of `federation-v2`.
- The url should be `https://raw.githubusercontent.com/kubernetes-sigs/federation-v2/master/charts`